src/components/AddPeopleAndRoles/OrgPerson.vue

```diff
@@ -433,14 +498,10 @@ export default class OrgPerson extends Mixins(EntityFilterMixin, CommonMixin) {
     }
   }
 
-  // *** FUTURE: fix or delete this (activeIndex is never -1 so this always returns false)
```

Comment:
This was incorrect. The active index is -1 for a new org/person, and >= 0 for editing.